packages/cdk/lib/constructs/engines/engine-job-definition.ts

```diff
@@ -19,8 +19,8 @@ export class EngineJobDefinition extends JobDefinition {
           AWS_METADATA_SERVICE_NUM_ATTEMPTS: "10",
           ...props.container.environment,
         },
-        memoryLimitMiB: props.container.memoryLimitMiB || 2048,
-        vcpus: props.container.vcpus || 1,
+        memoryLimitMiB: props.container.memoryLimitMiB || 4096,
```

Comment:
We should actually probably keep this at 1 vcpu and 4096 RAM. When I suggest 2 vCPU I had forgotten that these are Fargate jobs so it is possible for it to be 1. Sorry about that.

Reply:
np, changing it back.